client/SFTPClientGUI/SFTPClientGUI.py

```diff
@@ -0,0 +1,366 @@
+		col_date.pack_start(col_text, True)
+		col_date.add_attribute(col_text, 'text', 6)
+		col_date.set_sort_column_id(6)
+		self.treeview_local.append_column(col)
+		self.treeview_local.append_column(col_perm)
+		self.treeview_local.append_column(col_size)
+		self.treeview_local.append_column(col_date)
+
+		self.treeview_local.connect('row-expanded', self.signal_expand_row)
+		self.treeview_local.connect('row-collapsed', self.signal_collapse_row)
+		self._tvmodel = Gtk.TreeStore(str, Pixbuf, str, str, str, int, str)
+
+		self.local_hidden = True
+		self._get_popup_menu()
+
+	def _get_popup_menu(self):
+		self.treeview_local.connect('button_press_event', self._signal_treeview_button_pressed)
+
+		self.popup_menu = Gtk.Menu.new()
+		menu_item = Gtk.CheckMenuItem.new_with_label('Show Hidden Files')
+		menu_item.connect('toggled', self.signal_menu_toggled_hidden_files)
+		self.signal_menu_toggled_hidden_files = menu_item
+		self.popup_menu.append(menu_item)
+
+		menu_item = Gtk.MenuItem.new_with_label('Delete')
+		menu_item.connect('activate', self.signal_menu_activate_delete_prompt)
+		self.popup_menu.append(menu_item)
+		self.popup_menu.show_all()
+
+	def _signal_treeview_button_pressed(self, _, event):
+		if event.button == Gdk.BUTTON_SECONDARY:
+			self.popup_menu.popup(None, None, None, None, event.button, Gtk.get_current_event_time())
+			return True
+		return
+
+	def signal_expand_row(self, _, treeiter, treepath):
+		new_path = self._tvmodel[treeiter][2]  # pylint: disable=unsubscriptable-object
+		self.load_dirs(new_path, treeiter)
+		self._tvmodel.remove(self._tvmodel.iter_children(treeiter))
+
+	def signal_collapse_row(self, _, treeiter, treepath):
+		current = self._tvmodel.iter_children(treeiter)
+		while current:
+			self._tvmodel.remove(current)
+			current = self._tvmodel.iter_children(treeiter)
+		self._tvmodel.append(treeiter, [None, None, None, None, None, None, None])
+
+	def load_dirs(self, path, parent=None):
+		counter = 0
+		for name in self._yield_dir_list(path):
+			if self.local_hidden and name.startswith('.'):
+				continue
+			fullname = path + '/' + name
+			perm = self._check_perm(fullname)
+			raw_time = self._get_raw_time(fullname)
+			date_modified = '   ' + time.strftime('%y-%m-%d %H:%M:%S', time.localtime(raw_time))
+			is_folder = self._get_is_folder(fullname)
+			if is_folder:
+				icon = Gtk.IconTheme.get_default().load_icon('folder', 20, 0)
+				if perm[3] != 'r' and perm[5] != 'x':
+					icon = Gtk.IconTheme.get_default().load_icon('emblem-unreadable', 13, 0)
+				current = self._tvmodel.append(parent, (name, icon, fullname, perm, None, -1, date_modified))
+			else:
+				file_size = self._get_file_size(fullname)
+				hr_file_size = '   ' + boltons.strutils.bytes2human(file_size)
+				if perm[3] != 'r' and perm[5] != 'x':
+					icon = Gtk.IconTheme.get_default().load_icon('emblem-unreadable', 13, 0)
+				icon = Gtk.IconTheme.get_default().load_icon('empty', 12.5, 0)
```

Comment:
Apparently this icon isn't available on Fedora 24, I get this error:
```
Error Type: GLib.Error
Error Details: GLib.Error('Icon 'empty' not present in theme Adwaita', 'gtk-icon-theme-error-quark', 0)
Error UID: 2876e50f-ec96-4905-85ce-f19d8cd269a9
RPC Error: N/A (Not a remote RPC error)
King Phisher Version: 1.4.0-beta (rev: 8efea224)
Platform Version: Linux: Fedora 24 Twenty Four
Python Version: 3.5.1
Gtk Version: 3.20.6
Timezone: America/New_York

Thread Information:
 => MainThread (alive=True daemon=False)
    Thread-20 (alive=True daemon=True)
    Thread-19 (alive=True daemon=False)

Stack Trace:
Traceback (most recent call last):
  File "/home/steiner/.config/king-phisher/plugins/SFTPClientGUI.py", line 40, in signal_sftp_start
    manager = FileManager(target_file, self.application, ftp)  #pylint: disable=unused-variable
  File "/home/steiner/.config/king-phisher/plugins/SFTPClientGUI.py", line 317, in __init__
    self.local = LocalDirectory(self.builder, self.logger, upload_button)  # pylint: disable=unused-variable
  File "/home/steiner/.config/king-phisher/plugins/SFTPClientGUI.py", line 223, in __init__
    self.load_dirs(init_dir)
  File "/home/steiner/.config/king-phisher/plugins/SFTPClientGUI.py", line 180, in load_dirs
    icon = Gtk.IconTheme.get_default().load_icon('empty', 12.5, 0)
GLib.GError: gtk-icon-theme-error-quark: Icon 'empty' not present in theme Adwaita (0)
```

This icon will need to be changed, but at least it's only referenced in this one location now.